Makefile

```diff
@@ -26,6 +26,30 @@ APPLICATION_NAME = $(shell jq -r '.displayName' package.json)
 APPLICATION_DESCRIPTION = $(shell jq -r '.description' package.json)
 APPLICATION_COPYRIGHT = $(shell cat electron-builder.yml | shyaml get-value copyright)
 
+BINTRAY_ORGANIZATION = resin-io
+BINTRAY_REPOSITORY_DEBIAN = debian
+BINTRAY_REPOSITORY_REDHAT = redhat
+
+# ---------------------------------------------------------------------
+# Extra variables
+# ---------------------------------------------------------------------
+
+TARGET_ARCH_DEBIAN = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t debian)
+TARGET_ARCH_REDHAT = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t redhat)
+TARGET_ARCH_APPIMAGE = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t appimage)
+TARGET_ARCH_ELECTRON_BUILDER = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t electron-builder)
+TARGET_PLATFORM_PKG = $(shell ./scripts/build/target-convert.sh -r $(TARGET_PLATFORM) -t pkg)
+ENTRY_POINT_CLI = lib/cli/etcher.js
+ETCHER_CLI_BINARY = $(APPLICATION_NAME_LOWERCASE)
+ifeq ($(TARGET_PLATFORM),win32)
+ETCHER_CLI_BINARY = $(APPLICATION_NAME_LOWERCASE).exe
+endif
+
+PRODUCT_NAME = etcher
```

Comment:
Any reason for a separate `PRODUCT_NAME` variable, when (AFAICT) it's the same as `APPLICATION_NAME_LOWERCASE` ?